srtp/srtp.c

```diff
@@ -693,16 +694,20 @@ static srtp_err_status_t srtp_kdf_init(srtp_kdf_t *kdf,
         break;
     }
 
-    srtp_err_status_t stat;
-    stat = srtp_crypto_kernel_alloc_cipher(cipher_id, &kdf->cipher, key_len, 0);
-    if (stat)
-        return stat;
+    {
```

Comment:
VS 2010 complains about variables defined further down?  If so, I'd be OK with moving "stat" to the top of the function.  I think it's unfortunate to introduce indentation just to address that issue.  (Plus, I think it's better to define variables at the top, anyway.  So, I'd personally prefer moving it.)